### ISSUE TITLE
Estimation field is not necessarily a long value.

### ISSUE THREAD

**Issue by johnkoerner** [dapplo/Dapplo.Jira#18]
I am running in an environment where the estimation field may be null or a double value.  This currently causes issues with the `long` that is expected with the current implementation.  I propose that the `GetEstimation` method be rewritten as a generic, like this:

```
/// <summary>
///     Retrieve the estimation (story points) from the issue, this only works by using the BoardConfiguration
/// </summary>
/// <param name="boardConfiguration">BoardConfiguration</param>
/// <returns>long with estimation or 0 if nothing</returns>
public long GetEstimation(BoardConfiguration boardConfiguration)
{
    return GetEstimation<long>(boardConfiguration);
}

public T GetEstimation<T>(BoardConfiguration boardConfiguration) 
{
    if (boardConfiguration == null)
    {
        throw new ArgumentNullException(nameof(boardConfiguration));
    }
    var estimationCustomField = boardConfiguration.Estimation.Field.FieldId;
    if (!Fields.CustomFields.ContainsKey(estimationCustomField))
    {
        return default(T);
    }

    var result = Fields.CustomFields[estimationCustomField];

    return result == null ? default(T) : (T)result;
}
```

**Comment by Lakritzator** [comment]
I think I didn't really realize there are multiple ways of having estimations, thanks for pointing this out AND having a really sensible solution. I will update the project with "your" solution and let you know when this is available. (probably today).

Have a few more fixes to make.

Thanks!

**Comment by Lakritzator** [comment]
So, I just published 0.6.10 which has this change in it. I had to change some of the dependencies, due to bug #17 which could only be solved by doing so. I hope this doesn't break anything for you...

Let me know if this works for you.
